src/content/docs/aws/getting-started/installation.mdx

```diff
@@ -72,7 +72,8 @@ You may download the binary for your architecture using the link below:
 
 or use the following curl command:
 
-<Code code={`curl --output localstack-cli-${LOCALSTACK_VERSION}-darwin-amd64-onefile.tar.gz \n    --location https://github.com/localstack/localstack-cli/releases/download/v${LOCALSTACK_VERSION}/localstack-cli-${LOCALSTACK_VERSION}-darwin-amd64-onefile.tar.gz`} lang="bash" />
+<Code code={`curl --output ${LOCALSTACK_VERSION}-darwin-amd64-onefile.tar.gz \
```

Comment:
@joe4dev I've updated the hardcoded version numbers to instead say `${LOCALSTACK_VERSION}`. 

Let me know if it looks good now 😸